tutorials/financial-tick-data/index.md

```diff
@@ -38,6 +38,10 @@ This tutorial covers:
     [Twelve Data][twelve-data] into your TimescaleDB database.
 1.  [Query your dataset][financial-tick-query]: Create candlestick views, query
     the aggregated data, and visualize the data in Grafana.
+1.  [Ingest real-time websocket data][advanced-websocket]: Ingest real-time
```

Comment:
This makes it look like a third step in the existing tutorial, and I think it's actually a tutorial in its own right?

Reply:
Yes!!!